overlord/devicestate/handlers_test.go

```diff
@@ -111,6 +122,11 @@ func (s *deviceMgrSuite) TestSetModelHandlerNewRevision(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(fooState.Flags.Required, Equals, true)
 	c.Check(barState.Flags.Required, Equals, false)
+	// the kernel is no longer required
+	err = snapstate.Get(s.state, "pc-kernel", &barState)
```

Comment:
this reads a bit odd reusing barState but ok

Reply:
Indeed, sorry for this!